pallets/liquidation/src/lib.rs

```diff
@@ -114,7 +124,11 @@ pub mod pallet {
     }
 
     #[pallet::hooks]
-    impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+    impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T>
+    where
```

Comment:
do we really need these two trait bounds here? because you didn't use in hooks

Reply:
Yes, we need for calling `accrue_interest` which is under the trait bounds.